sql/testdata/order_by

```diff
@@ -118,3 +117,36 @@ SELECT * FROM t ORDER BY foo
 query error qualified name "a.b" not found
 SELECT a FROM t ORDER BY a.b
 ----
+
+statement ok
+CREATE TABLE abc (
+  a INT,
+  b INT,
+  c INT,
+  PRIMARY KEY (a, b, c),
+  CONSTRAINT bc INDEX (b, c),
+  CONSTRAINT ba INDEX (b, a)
+)
+
+statement ok
+INSERT INTO abc VALUES (1, 2, 3)
```

Comment:
it may be illustrative to add another row here to demonstrate that we scan forward (and sort in-memory) even when reverse order is requested